CPU(s):                          24
On-line CPU(s) list:             0-23
Model name:                      AMD Ryzen 9 5900X 12-Core Processor
Thread(s) per core:              2
Core(s) per socket:              12
CPU(s) scaling MHz:              62%
NUMA node0 CPU(s):               0-23
5.16.11-1-default
segul 0.19.0
v0.3.5
Benchmarking SEGUL
Dataset path: alignments/wu_2018_aa_loci/
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5584 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5460 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5620 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5520 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5708 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5740 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5556 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5460 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5640 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5620 100%
Dataset path: alignments/shen_2018_loci_aa/
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4964 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4968 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4776 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5028 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4612 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4664 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4896 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4916 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4708 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4760 100%
Benchmarking SEGUL ignore datatype
Dataset path: alignments/wu_2018_aa_loci/
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5828 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5464 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5664 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5528 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5760 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5460 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5580 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5556 80%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5652 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5628 100%
Dataset path: alignments/shen_2018_loci_aa/
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4896 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4776 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4904 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4756 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4900 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4728 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4772 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4840 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 5092 100%
error: unrecognized subcommand 'concat'

  tip: a similar subcommand exists: 'contig'
  tip: to pass 'concat' as a value, use 'segul -- concat'

Usage: segul [OPTIONS] <COMMAND>

For more information, try '--help'.
Command exited with non-zero status 2
0:00.00 4588 100%

Benchmarking AMAS
Dataset path: alignments/wu_2018_aa_loci/
0:01.72 1172740 1048%
0:01.76 1170624 1046%
0:01.73 1170740 1066%
Process ForkPoolWorker-8:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-12:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-21:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-9:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-2:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-15:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-24:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-20:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-1:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-23:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-13:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-17:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-18:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-7:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-14:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-10:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-6:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-3:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-22:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-5:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-4:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-11:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-19:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 351, in get
    with self._rlock:
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/synchronize.py", line 95, in __enter__
    return self._semlock.__enter__()
KeyboardInterrupt
Process ForkPoolWorker-16:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 110, in worker
    task = get()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/queues.py", line 352, in get
    res = self._reader.recv_bytes()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/connection.py", line 216, in recv_bytes
    buf = self._recv_bytes(maxlength)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/connection.py", line 407, in _recv_bytes
    buf = self._recv(4)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/connection.py", line 379, in _recv
    chunk = read(handle, remaining)
KeyboardInterrupt
Traceback (most recent call last):
  File "/home/hhandika/Program/AMAS.py", line 2075, in <module>
    main()
  File "/home/hhandika/Program/AMAS.py", line 2050, in main
    meta_aln.write_out("concat", kwargs["out_format"])
  File "/home/hhandika/Program/AMAS.py", line 1986, in write_out
    self.write_concat(file_format)    
  File "/home/hhandika/Program/AMAS.py", line 1920, in write_concat
    concatenated_alignment = self.get_concatenated(self.parsed_alignments)[0]
  File "/home/hhandika/Program/AMAS.py", line 1577, in get_concatenated
    if taxon not in all_taxa:
KeyboardInterrupt
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/util.py", line 277, in _run_finalizers
    finalizer()
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/util.py", line 201, in __call__
    res = self._callback(*self._args, **self._kwargs)
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 581, in _terminate_pool
    cls._help_stuff_finish(inqueue, task_handler, len(pool))
  File "/home/hhandika/anaconda3/lib/python3.7/multiprocessing/pool.py", line 566, in _help_stuff_finish
    inqueue._rlock.acquire()
KeyboardInterrupt
Command exited with non-zero status 1
0:02.34 359596 17%
